MainClass.cs

```diff
@@ -0,0 +1,149 @@
+        {
+            List<Item> allItems = new List<Item>();
+            foreach (Shelf shelf in refrigerator.shelves)
+            {
+                allItems.AddRange(shelf.items);
+            }
+
+            return allItems.OrderBy(item => item.expirationDate).ToList();
+        }
+
+        static List<Shelf> SortShelfBySpace(List<Shelf> shelves)
+        {
+            List<Shelf> sortedShelves = shelves.OrderByDescending(shelf => shelf.availableSpace).ToList();
+            return sortedShelves;
+
+        }
+        static List<Refrigerator> SortFridgeBySpace(List<Refrigerator> fridges)
+        {
+            List<Refrigerator> sortedFridges = fridges.OrderByDescending(fridge => fridge.GetAvailableSpace()).ToList();
+            return sortedFridges;
+
+        }
+
+
+
+        public static int Main()
+        {
+
+            Program program = new Program();
+            List<Refrigerator> refrigerators = program.createInfrastructure();
+            Console.Clear();
+            Refrigerator chosenFridge;
+
+            try
+            {
+
+                chosenFridge = program.ShowManual(refrigerators);
+                int option = program.UserInputOptions();
+
+                while (option != 100)
+                {
+
+                    switch (option)
+                    {
+
+                        case 1:
+                            Console.WriteLine(chosenFridge);
+                            break;
+                        case 2:
+                            Console.WriteLine("Available Space: ");
+                            Console.WriteLine(chosenFridge.GetAvailableSpace());
+
+                            break;
+                        case 3:
+                            try
+                            {
+                                Console.WriteLine();
+                                Item userItem = program.GetItemFromUser();
+                                chosenFridge.AddToRefrigerator(userItem);
+                            }
+                            catch (Exception ex)
+                            {
+                                Console.WriteLine(ex.Message);
+
+                            }
+                            break;
+                        case 4:
+                            try
+                            {
+                                Console.WriteLine("Enter Item ID");
+                                int Itemid = program.UserInputId();
+                                Item item = chosenFridge.RemoveFromRefrigerator(Itemid);
+                                Console.WriteLine(item);
+                            }
+                            catch (Exception ex)
+                            {
+                                Console.WriteLine(ex.Message);
+
+                            }
+                            break;
+                        case 5:
+                            chosenFridge.CleanRefrigerator();
```

Comment:
II write here feedback about all clean flow:
It's hard  @to understand because function names are not clear. 
```
case 5:
    chosenFridge.CleanRefrigerator();
    break;

Refrigerator:
public void CleanRefrigerator()
{
    List<Item> itemForRemove = new List<Item>();
    foreach (Shelf shelf in shelves)
    {
        var items = shelf.CleanShelf();//in this function you don't clean , you just return items
        if (items != null)
        {
            itemForRemove.AddRange(items);
        }
    }

    PrintCleanStatus(itemForRemove);// in this function you clean, but it's name say it just printing
}
```
It's make more sense to have CleanShelf that remove items and return removed items, than return a list and then for each Item you need to look on it in all shelves